x/farming/types/params_test.go

```diff
@@ -44,7 +44,7 @@ func TestParamsValidate(t *testing.T) {
 		{
 			"NegativeEpochDays",
 			func(params *types.Params) {
-				params.EpochDays = uint32(0)
+				params.NextEpochDays = uint32(0)
```

Comment:
The name of the test case says it is "NegativeEpochDays" but the case uses `0` for the value. Since `NextEpochDays` is a `uint32` field, we can rename NegativeEpochDays to ZeroEpochDays.

Reply:
I will rename it to `ZeroNextEpochDays`